### ISSUE TITLE
[Bug]: Chrome compatibility issue with expanding and collapsing video description

### ISSUE THREAD

**Issue by imightbelosthere** [tubearchivist/tubearchivist#327]
### Latest and Greatest

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).

### Operating System

Docker on Linux

## Describe the bug
Video description gets truncated and clicking on Show More doesn't really reveal more information.

## Steps To Reproduce
Download a video, Access it in Tube Archivist and click on Show More.

## Expected behavior
Getting the full video description just like in YT.
![tube_issue_show_more](https://user-images.githubusercontent.com/74044932/192738719-2e46648d-3f74-4185-9a6b-6e7cc99f2f38.png)

### Relevant log output

```shell
I don't have a log output but more than happy to get it if needed be - just need instructions on how and what to capture.
```

**Comment by bbilly1** [comment]
I can't observe the same behavior, for most of the videos I've checked, once you click on *show more* this expands the whole description. 

Please provide the link to that video to compare. Also verify in the API `/api/video/<video-id>/` of your instance and see if the `description` key there holds the whole description.

**Comment by imightbelosthere** [comment]
Hi @bbilly1,

Video ID is: CjABEnRg9NI

Indeed with the /api/video/video-id I can see the full description there...

Could this be a consequence of me upgrading from an old version? I've essentially upgraded the version on docker for the container and it did everything on its own. Maybe I've missed a step?

**Comment by bbilly1** [comment]
This works fine for me, once expanding the description, the whole description shows as on YouTube. The description is indexed correctly, as you can see in the API, but I did just test, on Chrome, this doesn't expand correctly, but works fine on Firefox. 

We'll have to look into this, thanks for raising the issue.

**Comment by imightbelosthere** [comment]
> This works fine for me, once expanding the description, the whole description shows as on YouTube. The description is indexed correctly, as you can see in the API, but I did just test, on Chrome, this doesn't expand correctly, but works fine on Firefox.
> 
Can confirm the behavior on Firefox and, of course, the issue with Edge (Chromium based browser).

Thanks for checking that.

**Comment by bbilly1** [comment]
@p0358 Do you have time to look into this?

**Comment by bbilly1** [comment]
That is now fixed in v0.2.3. Thanks for reporting!
